src/lib/github.js

```diff
@@ -31,31 +30,52 @@ export function getRepositories() {
   }
 }
 
+async function verifyRepoExists(repo) {
+  const apiUrl = getApiUrl(`/repos/${repo}`);
+  const response = await fetch(apiUrl);
+  debugger;
```

Comment:
`debugger;`

Reply:
Nice catch! Should have been caught by the linter though. We should look into that.

Reply:
Yeah, I wonder if it's because that lint rule was only added to `markdown-it-music`.